.github/workflows/new-releases-check.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         --
         config.json
         update
+        -dep:https://raw.githubusercontent.com/xamarin/GooglePlayServicesComponents/main/config.json
```

Comment:
Shouldn't that be AndroidX?

https://raw.githubusercontent.com/xamarin/AndroidX/main/config.json

Reply:
The dependencies live in the GPS repository, so we want to pull the `config.json` from there in order to find the versions needed.